examples/python3/useful/geometry.py

```diff
@@ -67,7 +67,7 @@ def calculate_frame_parameters(beamGeoParams):
     return frames_f, curv_abs_output_f, cable_position_f
 
 
-def generate_edge_list(cable3DPos: List[List[float]]) -> List[int]:
+def generate_edge_list(cable3DPos: List[List[float]]) -> list[list[int]]:
```

Comment:
What is the difference between List and list

Reply:
" `List` is a generic version of `list`. Useful for annotating return types.", List is more suited to types, although both work for types since 3.9. So the old version of the code works too. But you know all that, and that's why you asked, so I'm reverting to the old version :-).

Reply:
Actually I'm not sure knew it :)